sass/mosaico/_form.scss

```diff
@@ -0,0 +1,27 @@
+.crm-mosaico-page {
+  #inputSubject {
+    width: 80%;
+  }
+
```

Comment:
Fixed in a new commit 
https://github.com/compucorp/uk.co.vedaconsulting.mosaico/pull/4/commits/5635e5d993ba9e502acf3d9a9c0d0c75d3b32dca